src/vmm_mad/remotes/lib/lxd/svncterm_server/SConstruct

```diff
@@ -19,13 +19,14 @@ import os
 env = Environment()
 
 env['LIBS'] = ['-lz']
+nv['CFLAGS'] = ['-std=c99','-Wall']
```

Comment:
typo ?

Reply:
definitely, copy paste from my patch file.
do you want a new patch to fix it ?